packages/core/db/queries/getUser.ts

```diff
@@ -0,0 +1,9 @@
+import { eq } from "drizzle-orm";
+
+import { db } from "@/db/index";
+import { users } from "@/db/schema";
+
+export const getUser = async (userId: string) =>
```

Comment:
The deleUser uses the teamId too, not seems consistent, and if including the teamId too, then we have a getTeammates which are doing the same with a bit differnt sytax.

Reply:
Ur right but this query is only used for testing and because we are only going to use it inhouse anyway, the team id would be the same in each query.

I am happy to add the teamId in, but then the query params need to include it as well and it seems redundant as this is just for testing.

getTeammates is different because it gets everyone except that user so its not applicable.

Reply:
I'm okay to leave it; just my eyes stopped there; why are we leaving the team here and not the other places? Actually it's not needed at all atm. But that's makes sense don't need to complicate the endpoint with additional params

You are right about the getTeammates. That's my bad; there is a not(..